bundles/org.openhab.binding.yamahareceiver/src/main/resources/OH-INF/i18n/yamahareceiver.properties

```diff
@@ -179,3 +179,8 @@ channel-type.yamahareceiver.volume.label = Volume
 channel-type.yamahareceiver.volume.description = Set the volume level
 channel-type.yamahareceiver.volumeDB.label = Volume in dB
 channel-type.yamahareceiver.volumeDB.description = Set the volume level (dB)
+
+# channel types
+
+channel-type.yamahareceiver.surroundProgram.state.option.Neo:6 Music = Neo:6 Music
+channel-type.yamahareceiver.surroundProgram.state.option.Neo:6 Cinema = Neo:6 Cinema
```

Comment:
Please take care, there is a problem with Neo:6, the entries are doubled now.
To be honest, I don't know if the \ is required or not. Maybe @jlaur or @wborn can confirm ?

Reply:
Yes see https://github.com/openhab/openhab-addons/pull/11828

Reply:
Fixed.

Sorry, I did not check what the i18n plugin generated - I should probably don't trust it without checking what it does.